test/JDBC/upgrade/16_10/schedule

```diff
@@ -607,8 +607,9 @@ BABEL-NUMERIC_EXPR
 test_conv_float_to_varchar_char
 BABEL-5467
 BABEL_OBJECT_RESOLUTION_IN_ROUTINES
-test_constraint_like
+#test_constraint_like
```

Comment:
Nit: why are we disabling this test?

Reply:
We will re-enable it again when we have these changes in 16.latest else it will fail here.